x-pack/solutions/observability/plugins/profiling/public/components/frame_information_window/index.tsx

```diff
@@ -159,7 +159,9 @@ export function FrameInformationWindow({
           </EuiFlexGroup>
         </EuiFlexItem>
         <FrameInformationAIAssistant frame={frame} />
-        {showSymbolsStatus && symbolStatus !== FrameSymbolStatus.SYMBOLIZED ? (
+        {showSymbolsStatus &&
+        symbolStatus !== FrameSymbolStatus.SYMBOLIZED &&
+        frame.frameType !== FrameType.Root ? (
```

Comment:
https://github.com/elastic/elasticsearch/pull/119115 introduces two more grouping levels below the root frame (process name and thread name). These artificial frames all have a frame type >= 0x100. This change works here:
```suggestion
        frame.frameType < FrameType.Root ? (
```

Reply:
fixed here https://github.com/elastic/kibana/pull/209978/commits/cb6d02b5a0e9e77ef48e1adf7650443a5504ffcb